pages/projects.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <section>
+    <CommonGrid :projects="projectsList" data-cy="projects" />
+  </section>
+</template>
+
+<script>
+export default {
+  async asyncData({ $content }) {
+    const { projects } = await $content('projects').fetch()
+
+    const projectsList = projects.list.sort(
+      ({ name: nameA }, { name: nameB }) => nameA.localeCompare(nameB)
+    )
```

Comment:
💬 We could create a method `sortByName` and use it in here and `maintainers`. What do you think?

Reply:
Definitely, could be separated into a `utils` function, that said, I'm not confident if we need to order projects alphabetically.

Reply:
I think it's better ordering alphabetically than no other at all 👍